### ISSUE TITLE
Show chevron on also on:focus in the list

### ISSUE THREAD

**Issue by Bojhan** [TheIdentitySelector/thiss-js#265]
Currently we dont show the chevron when focusing on items in the list. 

Since we cannot currently rely on browsers to meet WCAG 2.2, this should also be triggered on:focus

<img width="546" alt="Screenshot 2024-01-12 at 10 11 56" src="https://github.com/TheIdentitySelector/thiss-ds-js/assets/209807/fa94abc2-66ff-4c83-a38a-fdef8c9110d1">

**Comment by Bojhan** [comment]
Fixed in staging
